Documentation/api-reference.md

```diff
@@ -35,35 +35,66 @@ public static String extensionVersion()
 String extensionVersion = Edge.extensionVersion();
 ```
 
+#### Kotlin
+
+##### Syntax
+```kotlin
+fun extensionVersion(): String
+```
+
+##### Examples
+```kotlin
+val extensionVersion = EdgeBridge.extensionVersion()
+```
+
 ------
 
 ### getLocationHint
 
-Gets the Edge Network location hint used in requests to the Adobe Experience Platform Edge Network. The Edge Network location hint may be used when building the URL for Adobe Experience Platform Edge Network requests to hint at the server cluster to use.
+Gets the Edge Network location hint used in requests to the Edge Network. The location hint may be used when building the URL for Edge Network requests to hint at the server cluster to use.
```

Comment:
can we keep `The Edge Network location hint` as it provides more context on which hint in particular will be used